server/src/main/java/info/doseamigos/meds/MedWebService.java

```diff
@@ -2,34 +2,110 @@
 
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.util.IOUtils;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
+import info.doseamigos.ClientRequestObject;
+import info.doseamigos.CommonModule;
+import info.doseamigos.amigousers.AmigoUser;
+import info.doseamigos.authusers.AccessTokenDeserializer;
+import info.doseamigos.authusers.AuthUser;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
 
 /**
  * Rest Handlers for Meds.
  */
 public class MedWebService {
 
     private MedService medService;
+    private ObjectMapper objectMapper;
 
     public MedWebService() {
         Injector injector = Guice.createInjector(
+            new CommonModule(),
             new MedGuiceModule()
         );
         this.medService = injector.getInstance(MedService.class);
+        this.objectMapper = injector.getInstance(ObjectMapper.class);
     }
 
     /**
      * POST/PUT call that either adds or updates an existing med.
-     * @param medInfo The med info coming in from the client.
+     * @param inputStream The request information coming in from the client.
      * @param context The LambdaContext.
      * @return The updated Med.
      */
-    public Med createOrUpdateNewMed(Med medInfo, Context context) {
+    public void createOrUpdateNewMed(
+        InputStream inputStream,
```

Comment:
Using StreamHandlers instead of PojoHandlers allows me to use my custom ObjectMapper.  I didn't know how to add custom deserializers to the default Amazon one, so this works.